overlord/export_test.go

```diff
@@ -53,3 +56,10 @@ func MockEnsureNext(o *Overlord, t time.Time) {
 func (o *Overlord) Engine() *StateEngine {
 	return o.stateEng
 }
+
+func MockStoreNew(new func(*store.Config, auth.AuthContext) *store.Store) (restore func()) {
```

Comment:
we are quite inconsistent about giving doc comments to export_test.go stuff, but I can add one